### ISSUE TITLE
attributes alias 

### ISSUE THREAD

**Issue by Bahramibh** [sequelize/sequelize#11645]
there is a bug in attributes alias 
on link : https://sequelize.org/master/manual/querying.html 
when i wana use attributes: [id , name , [lname , 'as last lname ']
its actualy work , but result goto dataValues 
not data 
i should map data before send to responce 
like this 
return result.map(r=>r.dataValues)

**Comment by papb** [comment]
Indeed, I was able to reproduce. [SSCCE](https://github.com/papb/sequelize-sscce):

```js
async function(createSequelizeInstance, log) {
    const { DataTypes } = require('sequelize');
    const sequelize = createSequelizeInstance();
    const Product = sequelize.define('product', { name: DataTypes.STRING });
    await sequelize.sync();
    await Product.create({ name: 'foo' });
    const found = await Product.findOne({
        attributes: [
            ['name', 'renamed']
        ]
    });
    log(found.renamed); // undefined
    log(found.dataValues.renamed); // 'foo'
    log(found.get('renamed')); // 'foo'
};
```

However I will consider this only a docs issue, not a bug per-se, because in fact the "technically correct" way to get a value from an instance is `instance.get('attributeName')`. The syntax `instance.attributeName` is a magical getter created from the originally existing attribute names.

I will clarify this in the docs, thanks!

**Comment by ephys** [comment]
This will be fixed by https://github.com/sequelize/sequelize/issues/14300, I'm relabeling it as a bug
